provider/datagen/tests/data/postcard/fingerprints.csv

```diff
@@ -6070,7 +6070,7 @@ relativetime/short/year@1, sr-Latn, 121B, ec453c97466f78e1
 relativetime/short/year@1, th, 131B, 956e97879a6bd7ad
 relativetime/short/year@1, tr, 107B, a182ac8d5087b3cb
 relativetime/short/year@1, und, 62B, 45e258268a1e1f9
-segmenter/dictionary/wl_ext@1, th, 224981B, f2d574736bb1a754
+segmenter/dictionary/wl_ext@1, th, 449959B, 1e470e1d1c880a7f
```

Comment:
Issue: The dictionary is doubling in size, despite there not being a diff on the Thai dictionary for 2 years. There is a bug somewhere. It looks like it is coming from ICU4C since the dictionary toml file also doubled in size.

Reply:
Good catch! We should probably output fingerprints for full baked data, we're flying blind for all the non-test locales.